main/wallet-backend.js

```diff
@@ -10,21 +10,15 @@ const { CoinType } = require('@glif/filecoin-address')
 const { strict: assert } = require('node:assert')
 const { Message } = require('@glif/filecoin-message')
 const { FilecoinNumber, BigNumber } = require('@glif/filecoin-number')
-const { request, gql } = require('graphql-request')
-const pMap = require('p-map')
+const { default: fetch } = require('node-fetch')
```

Comment:
Nitpick - feel free to ignore it.

I personally prefer to use `fetch` from `undici`. It's maintained by the Node.js project and shipped as the default `fetch` implementation from Node.js version 18.

After we upgrade to Electron 23, which ships Node.js 18 (https://github.com/filecoin-station/filecoin-station/pull/476), we should be able to use this built-in `fetch` implementation and can remove dependency on `undici` or `node-fetch`.

Maybe you can add a code comment or create a GH issue to remind us to start using the built-in `fetch` implementation after upgrading to Node.js 18?